.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "editor.trimAutoWhitespace": false,
+    "files.trimTrailingWhitespace": false,
```

Comment:
BTW, would you consider trimming trailing whitespace in openfl/lime code? I find it very annoying to maintain the current aligned tabs.

Reply:
Could you give an example of trailing whitespace? Do you mean between lines, or on the same line?

Reply:
Oh, so the editor is removing whitespace when saving a file, as compared to while editing?

Reply:
Trailing whitespace is a whitespace (tab or space) to the right of a non-whitespace character, e.g.

```
someCall()<space><space>
```

in OpenFL/Lime codebases specifically there are a lot of whitespaces on the empty lines, like
```
<space>function f() {
<space><space>
<space><space>doSomeCall();<space>
<space><space>
<space>}
```

trimming trailing whitespace means that the text above will be saved as:
```
<space>function f() {

<space><space>doSomeCall();

<space>}
```

Reply:
(the empty line themselves are preserved of course, but they will be empty instead of having a number of whitespace characters)